README.md

```diff
@@ -11,11 +11,3 @@ For other meeting notes, see below:
 | TC39-TG3: Security                                           | [tc39/tg3/meetings][tg3-notes]                  | [tc39/tg3][tg3-agenda]                     |
 | Incubator Meetings                                           | [tc39/incubator-agendas/notes][incubator-notes] | [tc39/incubator-agendas][incubator-agenda] |
 
-[tg1-agenda]: https://github.com/tc39/agendas#tc39-meeting-agendas
-[tg2-agenda]: https://github.com/tc39/ecma402/projects/2
-[tg3-agenda]: https://github.com/tc39/tg3#agenda
-[incubator-agenda]: https://github.com/tc39/incubator-agendas#tc39-incubator-call-agendas
-[tg1-notes]: https://github.com/tc39/notes/tree/main/meetings
-[tg2-notes]: https://github.com/tc39/ecma402/tree/main/meetings
-[tg3-notes]: https://github.com/tc39/tg3/tree/main/meetings
-[incubator-notes]: https://github.com/tc39/incubator-agendas/tree/main/notes
```

Comment:
this will break the links in the readme

Reply:
I can't remember which it was, but there is some formatter that erroneously does this automatically.  might be `prettier`

Reply:
ugh yeah it was one of the lint automatic fixes